docs/src/main/sphinx/connector/jdbc-common-configurations.fragment

```diff
@@ -33,3 +33,7 @@ connector:
       Do not change this setting from the default. Non-default values may
       negatively impact performance.
     - 1000
+  * - ``join-pushdown.enabled``
+    - Enable :ref:`join pushdown <join-pushdown>`. Join pushdown is disabled by
+      default. Enabling this may negatively impact performance for some queries.
```

Comment:
Should I add this?

```rst
It can also be enabled using by setting the :doc:`catalog session property </sql/set-session>` ``join_pushdown_enabled`` to ``true``.
```

Reply:
Would condense into "Can also be enabled on a per-session basis with the ``join_pushdown_enabled`` :doc:`session property </sql/set-session>`.